src/models/user-model.js

```diff
@@ -1,48 +1,48 @@
-import {v4 as uuidv4} from 'uuid';
-
-const users = [];
-
-module.exports = class User {
-    constructor(login, password, age) {
-        this.id = uuidv4();
-        this.login = login;
-        this.password = password;
-        this.age = age;
-        this.isDeleted = false;
-    }
-
-    save() {
-        users.push(this);
+import Sequelize from "sequelize";
+import database from "../config/database";
+import {v4 as uuidv4} from "uuid";
+
+const users = [
+    {
+        id: uuidv4(),
+        login: 'asd',
+        password: 'asd',
+        age: '12'
+    },
+    {
+        id: uuidv4(),
+        login: 'asdasdasd',
+        password: 'asdasdasd',
+        age: '134232'
+    },
+];
+
+const User = database.define('user', {
+    id: {
+        type: Sequelize.STRING,
+        primaryKey: true,
+        allowNull: false,
+    },
+    login: {
+        type: Sequelize.STRING,
+    },
+    password: {
+        type: Sequelize.STRING,
+        allowNull: false,
+    },
+    age: {
+        type: Sequelize.STRING
     }
+});
 
-    static getById(id) {
-        return users.find(user => user.id === id);
-    }
-
-    static getAutoSuggestUsers(loginSubstring, limit) {
-        let result = users.filter(user => user.login.includes(loginSubstring));
-        result.sort();
-        result.slice(0, limit - 1);
-
-        return result;
+User.beforeValidate(async (user, options) => {
```

Comment:
async is not needed here. It is needed to use `async` only if function does smth asynchronous